zeebe/exporters/operate-exporter/src/main/java/io/camunda/zeebe/operate/exporter/ExportBatchWriter.java

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.zeebe.operate.exporter;
+
+import io.camunda.operate.entities.OperateEntity;
+import io.camunda.operate.entities.dmn.DecisionInstanceEntity;
+import io.camunda.operate.exceptions.PersistenceException;
+import io.camunda.operate.store.elasticsearch.NewElasticsearchBatchRequest;
+import io.camunda.operate.util.CollectionUtil;
+import io.camunda.operate.util.Tuple;
+import io.camunda.zeebe.operate.exporter.handlers.ExportHandler;
+import io.camunda.zeebe.protocol.record.Record;
+import io.camunda.zeebe.protocol.record.ValueType;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Accumulates Operate entities of different types and provide the method to flush them in batch.
+ */
+public class ExportBatchWriter {
+
+  // static
```

Comment:
Can you expand or clarify this 'static' comment (and the 'dynamic' one below)? It's unclear exactly what it's referring to and 'static' can be confusing since that's a java keyword and the variable is not static.